src/wp-includes/deprecated.php

```diff
@@ -6372,7 +6372,7 @@ function wp_get_global_styles_custom_css() {
  */
 function wp_enqueue_global_styles_custom_css() {
 	_deprecated_function( __FUNCTION__, '6.7.0', 'wp_enqueue_global_styles' );
-	if ( ! wp_is_block_theme() ) {
+	if ( ! wp_is_block_theme() || is_customize_preview() ) {
```

Comment:
Actually, I think this change can be undone. It's a deprecated function anyway so it is not used. And the logic here is not right as it should rather be doing something like this:

```php
$custom_css = '';
if ( ! is_customize_preview() ) {
	// Don't enqueue Customizer's custom CSS separately.
	remove_action( 'wp_head', 'wp_custom_css_cb', 101 );

	$custom_css = wp_get_custom_css();
}

$custom_css .= wp_get_global_styles_custom_css();
```

```suggestion
	if ( ! wp_is_block_theme() ) {
```

But again, given that this function is deprecated it doesn't seem necessary to make this change.